.github/workflows/bench.yml

```diff
@@ -0,0 +1,100 @@
+name: model_signing benchmarks
+on:
+  issue_comment:
+    types: [created]
```

Comment:
Right now, this benchmark is triggered by `/bench` only. Perhaps a `tag` and `workflow_dispatch` trigger too?